test_review.py

```diff
@@ -0,0 +1,2 @@
+print("hello dev!")
+print("please review my change
```

Comment:
Kindly check your second line and make sure the syntax are same like the first one